kale/embed/mpca.py

```diff
@@ -49,16 +49,16 @@ def _check_shape(x, shape_):
         raise ValueError(error_msg)
 
 
-def _check_dim_shape(x, n_dim, shape_):
+def _check_order_shape(x, n_dims, shape_):
     """Check whether the order of the input data is consistent with the given value of order
```

Comment:
Google "the order of the input data" to see what it means in most contexts (it means the ordering, i.e. how they are ordered"

Reply:
@sz144 "check_order_shape" is weird. You are checking the shape of the tensor rather than the shape of the order. So "check_tensor_shape" may be better.